x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/painless/risk_scoring_reduce.painless

```diff
@@ -0,0 +1,35 @@
+Map results = new HashMap();
+results['notes'] = [];
+results['category_1_score'] = 0.0;
+results['category_1_count'] = 0;
+results['risk_inputs'] = [];
+results['score'] = 0.0;
+
+List inputs = [];
+for (state in states) {
+  inputs.addAll(state.inputs)
```

Comment:
Does it make sense? 🤔 
I am not sure, but it could duplicate the memory necessary for storing all inputs. An alternative would be removing this line and coping the inputs one by one inside the loop.

Reply:
I have fallen back to using states[0] for now and added comment